Front/src/app/page.tsx

```diff
@@ -0,0 +1,133 @@
+"use client";
+import { Grid, Button, Divider, Typography, Box} from '@mui/material'
+import styles from './page.module.css'
+import { useState } from 'react'
+
+
+export default function Home() {
+
+  const [data, setData] = useState< Array<{id:number, nombre: string, ingredientes: string[]}>>([]);
+
+  // <Array <String | number> >([])
+
+  // const Datos= () => {
+  //   fetch('http://localhost:3000/api/data')
+  //   .then( r => r.json())
+  //   .then( data => {
+  //     const sumarDatos = data.Lista.map((n:number) => n + 1);
+  //     setData([data.Saludo, sumarDatos])
+  //   })
+  //   .catch(error => console.error('Error fetching data', error))
+  // }
+
+  const getComida = () => {
+    fetch('http://localhost:3000/api/comidas')
+    .then(r => r.json())
+    .then(data => setData(data.comidas))
+    .catch(err => console.log('Error',err))
+  }
+
```

Comment:
This kind of logic can be exported into its own files to keep one the Single Responsability  principle of the SOLID principless